python/oneflow/nn/graph/block.py

```diff
@@ -679,13 +740,18 @@ def __getitem__(self, idx):
 
 class ParameterDictBlock(get_para_dict(ModuleBlock)):
     def __init__(
-        self, prefix: str = "", name: str = "", origin: ParameterDict = None,
+        self,
+        prefix: str = "",
+        name: str = "",
+        origin: ParameterDict = None,
+        belonged_graph: weakref.ProxyTypes = None,
     ):
         super().__init__()
         self._name_prefix = prefix
         self._name = name
         self.set_origin(origin)
```

Comment:
`self._belonged_graph = belonged_graph` should be moved before `set_origin` to get belonged_graph in sub_module